setup.py

```diff
@@ -273,7 +273,7 @@ def consume_arg(name):
     if cython_only:
         sys.exit(0)
 
-no_compilation = any(x in ['lint', 'format', 'docs'] for x in sys.argv)
+no_compilation = any(x in ('docs') for x in sys.argv)
```

Comment:
```suggestion
no_compilation = 'docs' in sys.argv
```
simplified the expression